src/liquid/utils/cloneAttributes.ts

```diff
@@ -1,3 +1,7 @@
+function isFalsy(value: string | null | undefined) {
+  return value === undefined || value === null || value === 'false'
```

Comment:
Wouldn't this also affect aria-disabled, for example, and other attributes, where an explicit "false" value might be intended?

Reply:
Should be no problem. `aria-disabled="false"` will be treated as if it wasn't there.